tests/kernel/install_kernel_flavor.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Install custom kernel and remove kernel-defaul
+# Maintainer: QE Kernel <kernel-qa@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils qw(zypper_call);
+
+sub run {
+    select_serial_terminal;
+
+    my $kernel_flavor = get_var('KERNEL_FLAVOR');
```

Comment:
maybe `get_required_var('KERNEL_FLAVOR');` and then avoid if in `zypper_call` line?